vendor/sigs.k8s.io/kustomize/api/internal/git/cloner.go

```diff
@@ -22,15 +22,11 @@ func ClonerUsingGitExec(repoSpec *RepoSpec) error {
 	if err = r.run("init"); err != nil {
 		return err
 	}
-	if err = r.run(
-		"remote", "add", "origin", repoSpec.CloneSpec()); err != nil {
-		return err
-	}
 	ref := "HEAD"
 	if repoSpec.Ref != "" {
 		ref = repoSpec.Ref
 	}
-	if err = r.run("fetch", "--depth=1", "origin", ref); err != nil {
+	if err = r.run("fetch", "--depth=1", repoSpec.CloneSpec(), ref); err != nil {
```

Comment:
previously, adding the clonespec as an origin was unambiguous. inlining it here, will `git` always interpret this arg as a remote, or is there any value this could hold that git would misinterpret as something else?